src/Compilers/Core/CodeAnalysisTest/Analyzers/AnalyzerFileReferenceAppDomainTests.cs

```diff
@@ -118,7 +120,8 @@ public class TestAnalyzer : DiagnosticAnalyzer
         private static Assembly OnResolve(object sender, ResolveEventArgs e)
         {
             Console.WriteLine($"Resolve in {AppDomain.CurrentDomain.Id} for {e.Name}");
-            return null;
+            return null; // Question: How nullable is enabled, and return type is `Assembly` not `Assembly?` without any CI build errors?
```

Comment:
I know why. It's subtle. Roslyn is configured to only give nullable warnings for netcoreapp3.1 and up target frameworks. This is because netcoreapp3.1 has nullable annotations on the standard library, thus the warnings given for it will be more accurate. Since all this code is only included for NET472 we just don't give nullable warnings for it.

I'm guessing this event is added to debug tests. I would just leave it unchanged.